### ISSUE TITLE
XP Globes from XP Globe plugin jumping up and down

### ISSUE THREAD

**Issue by Orochi88** [runelite/runelite#14171]
Currently, the XP globes move below the max total xp while an xp drop happens even when it's hidden. This causes them to bounce up and down. See attached file for reference.

![XP_Globe_displacement](https://user-images.githubusercontent.com/91033808/133944371-fa1f35d7-ce6f-49a4-a53f-3a2632da9ad0.gif)

**Comment by Alexsuperfly** [comment]
i believe this is fixed with fc183c13afb91e7715e75e9d0205e380e409dccf

**Comment by Adam-** [comment]
Is your xptracker config counter=off bar=off duration=something non permanent? If so this is probably fixed as Alex said in fc183c13afb91e7715e75e9d0205e380e409dccf

**Comment by Orochi88** [comment]
Yes, those are my settings for it. So I just have to wait for the update then? 👍

**Comment by Adam-** [comment]
Yes, I think so.
